src/identifiers/logic.py

```diff
@@ -344,6 +341,49 @@ def render_doi_from_pattern(article):
     return '{0}/{1}'.format(doi_prefix, doi_suffix)
 
 
+def preview_registration_information(article):
+    """
+    Generates a rudimentary printout of metadata
+    for proofing by the end user before attempting
+    to register the DOI with Crossref.
+    """
+    if article.journal.use_crossref:
+        crossref_context = create_crossref_context(article)
+
+        # FrozenAuthors do not exist yet
+        # crossref_context['authors'] = article.authors.all()
+
+        exclude = ['batch_id', 'timestamp', 'depositor_email', 'registrant',
+                   'now', 'timestamp_suffix', 'print_issn', 'citation_list']
+        for k in exclude:
+            crossref_context.pop(k)
+
+        metadata_printout = 'We will try to send the following ' \
+                            'provisional metadata to Crossref:<br><br>'
+
+        for k, v in crossref_context.items():
+            if k == 'authors':
+                i = 0
+                for author in crossref_context['authors']:
+                    i += 1
```

Comment:
you can use `enumerate` to avoid this pattern in python:

`for idx, author in enumerate(crossref_context["authors"]):`